dvc/tree/base.py

```diff
@@ -226,20 +226,7 @@ def _getsize(self, path_info):
     def getsize(self, path_info):
         if self.isfile(path_info):
```

Comment:
This will waste an API call, no need for it.

Reply:
What about calling it `get_file_size`?

Reply:
@isidentical No need, `getsize` is always called for a particular file. Like os.getsize and like https://github.com/intake/filesystem_spec/blob/master/fsspec/spec.py states (our long-term goal).